### ISSUE TITLE
Copy of Implement Accessibility Feedback for Missing 5103 Link & Claim Certification 

### ISSUE THREAD

**Issue by Kate-allbee** [department-of-veterans-affairs/va.gov-team#82655]
## Epic Description

Implementing Accessibility feedback from Design intent meeting for [Collaboration Cycle for 5103 link / Claim certification
](https://github.com/department-of-veterans-affairs/va.gov-team/issues/82160)

Scope - 

[Accessibility Feedback ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/82630)

**Should:** If possible, explore moving the relevant 5103 text from the PDF into an HTML page. Even if it's not in plain language, a web page is going to be a better user experience than a PDF for most assistive technology users. It also avoids the confusion of opening a PDF version of the form you're currently working on, and users potentially thinking they need to do both.

 **Should:** If it's not possible to move text to a webpage and you need to link to the PDF, work with a content person from CAIA to come up with clearer link text. "Download VA Form 21-526EZ" follows the style guide and is appropriate in situations where users are downloading a form that they'll need to complete. But in this case they'll just be referencing information the form. Link text describing the intended use case should help guide the user and hopefully minimize confusion, eg. "Review Evidence Tables on page 4 of VA Form 21-526EZ" or something like that.

 **Consider:** It may be possible to [link directly to the page of the PDF](https://helpx.adobe.com/acrobat/kb/link-html-pdf-page-acrobat.html) that you want users to read. That may also reduce confusion.
 
**Statement of Truth Component Feedback**
**Should:** Something to think about as you refine this, which I explained in a very muddled way in the meeting. When a screen reader user is going through a form workflow, it would be unusual for them to have their screen reader announce all of the text on the page from start to finish. Instead, most screen reader users will be tabbing between each input of the form and just listening to the labels associated with the input --- which means they miss any text on the page not associated with the input.

We ran into this when another team had must-read legal text with their statement of truth at the end of the form, where users were likely to miss that must-read text. The solution in the statement of truth component was to create a prop named`inputMessageAriaDescribedy`, which uses an `aria-describedby` attribute to make sure all of the relevant legal text gets announced when you tab to the name input.

The challenge for your Claim Certification and Signature section is that `inputMessageAriaDescribedy` only accepts a text string. Links don't get preserved, they're just announced as plain text. Additionally, some screen readers stop reading `aria-describedby` text when it's too long, and I think your text is probably right around that cutoff point.

Now, the links will be tab stops for screen reader users before they hit the name input, so you can assume screen reader users will have heard and had the chance to follow the "Review a summary of evidence" and "Download Form 21-526EZ" links if they choose to. But that will occur before they're likely to hear the legal text.

When you finalize the legal text that appears here, make sure you think about what version of that text you're going to include in the `inputMessageAriaDescribedy` prop --- whether that means editing both the visible text and the `inputMessageAriaDescribedy` text so the same text can work in both places, or providing an equivalent but grammatically different version in the `inputMessageAriaDescribedy` text.

---
## Tasks

- [ ] Confirm with team what exact legal text needs to go in the claim certification component (Do we need to work with CAIA on this?)
- [ ] Come to a decision about creating an html page for legal text instead of the link to download the pdf. How does this impact timeline?
- [ ] If we do keep the PDF, explore the option to link to a direct page within a pdf and revise link copy to be more specific about action being taken

## Acceptance Criteria

- [ ] Consensus on path forward re: pdf download link or html page
- [ ] Copy and links revised for Statement of Truth component

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)
